Dockerfile

```diff
@@ -1,9 +1,10 @@
-FROM ubuntu:18.04
+FROM ubuntu:20.04
```

Comment:
I quite like it, because it allows testing for a recent ubuntu version. Depending what @morphis has in mind for https://github.com/anbox/anbox/pull/1490, we could check _both_ versions at the same time.

Reply:
We should follow what the snap is doing here for now. There are some problems with building the snap in a github action I need to look into. Once we have that solved we can maybe even move to core20 in the snap as a base and switch back here too.

Reply:
Snap updated, I think we can proceed.